tpot/base.py

```diff
@@ -141,10 +141,14 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
             Random number generator seed for TPOT. Use this to make sure
             that TPOT will give you the same results each time you run it
             against the same data set with that seed.
-        config_dict: dictionary (default: None)
-            Configuration dictionary for customizing the operators and parameters that
-            TPOT uses in the optimization process.
-            For examples, see config_regressor.py and config_classifier.py
+        config_dict: dictionary or string (default: None)
+            Dictionary:
```

Comment:
I think for interactive mode, we use python dictionary instead of file path, but for command line mode, we set TPOT use to take file path. Should we change that?